app/assets/stylesheets/mortgage_calculator/components/_stamp-duty-calc.scss

```diff
@@ -0,0 +1,146 @@
+.step_one {
+  margin-top: 0;
+
+  @include respond-to($mortgagecalc_mq-s) {
+    margin-top: $baseline-unit*5;
+  }
+}
+
+.stamp-duty {
+  border: solid #999999 1px;
+  overflow: hidden;
+  padding: 0 1.3rem;
```

Comment:
We've updated the padding with $baseline-unit.

It'd be great to have a chat about what baseline-unit is for and how it differs from rem.